x-pack/test/functional/apps/canvas/custom_elements.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function canvasSmokeTest({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+  const browser = getService('browser');
+  const retry = getService('retry');
+  const PageObjects = getPageObjects(['canvas', 'common']);
+  const find = getService('find');
+
+  describe('custom elements', () => {
+    before(async () => {
+      // init data
+      await esArchiver.loadIfNeeded('logstash_functional');
+      await esArchiver.load('canvas/default');
+
+      // load test workpad
+      await PageObjects.common.navigateToApp('canvas', {
+        hash: '/workpad/workpad-1705f884-6224-47de-ba49-ca224fe6ec31/page/1',
+      });
+    });
+
+    it('creates a custom element from an element when prompted', async () => {
+      // find the first workpad element (a markdown element) and click it to select it
+      const elements = await testSubjects.findAll(
+        'canvasWorkpadPage > canvasWorkpadPageElementContent'
+      );
+      const mdElem = elements[0];
+      await mdElem.click();
```

Comment:
To click the first element you can simply use:
`testSubjects.click('canvasWorkpadPage > canvasWorkpadPageElementContent')`

it will find the first element in DOM that matches the locator and click it.